modules/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -710,6 +710,7 @@ OptionalBytesReference compareAndExchangeRegister(
                         Storage.BlobTargetOption.generationMatch()
                     )
                 );
+                stats.trackPostOperation();
```

Comment:
This was the only operation that wasn't being counted I think. This change was the reason for starting these tests.

Reply:
Now tested